src/prowjobsscraper/main.py

```diff
@@ -58,9 +58,19 @@ def main() -> None:
     else:
         usages_scrape_start_time = usages_scrape_end_time - relativedelta(months=1)
 
+    equinix_projects_credentials = [
```

Comment:
why we need different creds for the two projects?

Reply:
I thought it would make more sense to use read-only related project tokens. Also I am not sure it is possiblr to generate a token for both of the projects. I can change it if it is possible though.

Reply:
There is an issue with personal tokens (not project specific) I am in touch with Equinix support. I will fix that as soon as they fix it